src/jaegertracing/TracerTest.cpp

```diff
@@ -218,14 +219,15 @@ TEST(Tracer, testTracer)
             ->StartSpanWithOptions("test-span-with-default-system-timestamp",
                                    options)
             .release()));
+    /*
```

Comment:
please add a comment, or better a link to an issue describing why this test is being disabled

Reply:
ok. Issue inserted #167 and issue referenced in the code.